README.md

```diff
@@ -25,7 +25,7 @@ public class Main {
 }
 ```
 **WARNING!!!** Path to your project must contain only english letters. Also, it mustn't contain spaces. In other case `Injector` won't work correctly.
-- Your table should be named `books` and contain these columns: `id`, `title`, `price`.
+- Your table should be named `book` and contain these columns: `id`, `title`, `price`.
```

Comment:
revert

Reply:
but i have table dook with columns: id, title and price

Reply:
according to the naming convention, table in DB should be plural